indexing/src/main/resources/org/allenai/common/indexing/indexing.conf

```diff
@@ -75,6 +75,67 @@ org.allenai.common.indexing.paragraph_base: ${org.allenai.common.indexing.base}
   }
 }
 
+org.allenai.common.indexing.qa_base: ${org.allenai.common.indexing.base} {
+  elasticSearch: {
+    indexType: "sentence"
```

Comment:
I'd imagine you'd want to change the type to "question" or something similar?  That would allow this type of document to coexist with existing corpora in the same index, if that's something we ever want to do.